pkgs/data/machine-learning/resnet/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenvNoCC, fetchurl }:
+let
+  srcs = {
+    resnet50 = fetchurl {
+      url = "http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz";
+      sha256 = "0wqw22aj5pbrhhbb13nh8r26zkj3dkq1ajrhvx4gczly50jxr84i";
+    };
+    resnet101 = fetchurl {
+      url = "http://download.tensorflow.org/models/resnet_v1_101_2016_08_28.tar.gz";
+      sha256 = "068sl5x9z05j1wagfjwz3yrxnk1r8ikb8wrx931ckrsnbv7kz0ax";
+    };
+  };
+in
+  stdenvNoCC.mkDerivation rec {
+    pname = "resnet";
+    version = "2016-08-28";
+    installPhase = ''
```

```suggestion
    buildCommand = ''
```
and can get rid of `phases`

---

Or could use `runCommand` instead